README.md

```diff
@@ -125,6 +125,9 @@ crates through the gateway.
 
 ## Examples
 
+Add this to your `main.rs`, and make sure to add your bot token to your
```

Comment:
I disagree with calling this "adding" to `main.rs` since this is a `main.rs` template and not an addition.

Reply:
I mostly added this line to mirror the text that is added above the dependency section. But neither really add a lot, so I can remove both if you think thay aren't important.

Reply:
If I were just to change the text, what would you recommend?

Reply:
Something along the lines of "The following example is a good template for bootstrapping a new bot using twilight's HTTP and gateway clients alongside its in-memory based cache. In order to run this, put the following inside the `main.rs` file and set the `DISCORD_TOKEN` enviroment variable to your bot's token."

The other maintainers might have different opinions though, so this is just my suggestion